configure/README.md

```diff
@@ -5,3 +5,30 @@ Scripts used for configuring server nodes
 ## `nginx`
 
 nginx setup. Derived from [`whatwg/misc-server`](https://github.com/whatwg/misc-server)
+
+## `vscode`
```

Comment:
is this section necessary to add in addition to the config file?

if so, does it really belong in the main README or could it go into a separate doc page?

Reply:
Hmmm @brandondees @rianby64 you both are right. It IS part of "configuration" per say. The thought was anything "configure" related stuff goes in `/configure`. But that is more internal dev related (perhaps shouldn't be?). IMHO this documentation should be as close to introduction as possible. Perhaps an IDE section? /cc @tmornini 

In addendum i'll add what I did for VIM.

Thanks for this @rianby64